MSAL/src/native_auth/controllers/factories/MSALNativeAuthResultFactory.swift

```diff
@@ -71,33 +71,31 @@ final class MSALNativeAuthResultFactory: MSALNativeAuthResultBuildable {
                                 accountClaims: jsonDictionary)
     }
 
-    func makeAuthTokens(tokenResult: MSIDTokenResult, context: MSIDRequestContext) -> MSALNativeAuthTokens? {
+    func makeRawIdToken(tokenResult: MSIDTokenResult, context: MSIDRequestContext) -> String? {
         guard let refreshToken = tokenResult.refreshToken as? MSIDRefreshToken else {
```

Comment:
We don't need this guard block. Once removing this block I think we don't this factory method anymore

Reply:
Done.